internal/operators/manager.go

```diff
@@ -24,7 +26,7 @@ import (
 	"github.com/thoas/go-funk"
 )
 
-const customManifestFile = "custom_manifests.json"
+const controllerManifestFile = "custom_manifests.json"
```

Comment:
Renamed "custom manifest" to controller manifest, as it was confusing to have 2 customs manifests.
They are now "controller manifest" and "install manifest"